convert2rhel/backup/__init__.py

```diff
@@ -70,6 +70,13 @@ def push(self, restorable):
         if not isinstance(restorable, RestorableChange):
             raise TypeError("`%s` is not a RestorableChange object" % restorable)
 
+        # check if the restorable is already backed up
+        # if it is, we skip it
+        for r in self._restorables:
+            if r == restorable:
+                loggerinst.debug("Skipping: has already been backed up")
```

Comment:
```suggestion
                loggerinst.debug("Skipping: %s has already been backed up" % r.filepath)
```
I tested it like this and it works great, the message looks like `DEBUG - Skipping: /etc/security/limits.conf has already been backed up` and as you said, we only have the `__eq__` for restorablefile, so it's safe to use.

Reply:
We can't use `r.filepath` as not all restorables have that. I think RFE for the future is to implement a `__str__()`

Reply:
Me and Adam came up with getting the class name which has now been added